src/app/features/tech-record/components/change-vehicle-type/change-vehicle-type.component.html

```diff
@@ -0,0 +1,59 @@
+<div *ngIf="vehicleTechRecord">
+  <br />
+  <br />
+  <h2 class="govuk-heading-l">
+    {{ vehicleMakeAndModel }}
+  </h2>
+
+  <dl class="govuk-summary-list">
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Vehicle type</dt>
+      <dd id="vehicleType" class="govuk-summary-list__value">
+        {{ currentTechRecord?.vehicleType ?? '' | uppercase | defaultNullOrEmpty }}
+      </dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">VRM</dt>
+      <app-number-plate *ngIf="currentVrm" [vrm]="currentVrm"></app-number-plate>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">VIN/chassis number</dt>
+      <dd id="vehicleType" class="govuk-summary-list__value">{{ vehicleTechRecord.vin | defaultNullOrEmpty }}</dd>
+    </div>
+  </dl>
+
+  <form [formGroup]="form">
+    <app-select
+      #selectVehicleType
+      [label]="form.meta.children![0].label"
+      [formControlName]="form.meta.children![0].name"
+      [options]="vehicleTypeOptions"
+      (ngModelChange)="ngOnChanges()"
+    ></app-select>
+
+    <div *ngIf="selectVehicleType.value !== vehicleTechRecord!.techRecord[0].vehicleType && selectVehicleType.value !== ''">
+      <h1 class="govuk-heading-xl">Are you sure you want to change this Vehicle type?</h1>
+
+      <span class="govuk-body"
+        >You are about to change a {{ vehicleTechRecord!.techRecord[0].vehicleType | uppercase }} to a
+        {{ selectVehicleType.value | uppercase }} provisional technical record.</span
+      >
+
+      <br />
+      <br />
+    </div>
+
+    <div style="display: flex; justify-content: space-between; align-items: center">
+      <button
+        class="govuk-button"
+        data-module="govuk-button"
+        type="submit"
+        enterkeyhint="change vehicle type"
+        (click)="handleSubmitNewVehicleType(selectVehicleType.value)"
+      >
+        Confirm and continue
+      </button>
+      <a class="govuk-link margin-right" style="cursor: pointer" data-module="govuk-button" (click)="navigateBack()">Back</a>
```

Comment:
As discussed, `<a>` tags should have `[href]` or `[routerLink]`

Reply:
Changed to app-button with design="link"